vaccine_feed_ingest/stages/ingest.py

```diff
@@ -340,7 +340,18 @@ def run_enrich(
             enrich_output_dir,
         )
 
-        success = enrichment.enrich_locations(enrich_input_dir, enrich_output_dir)
+        enrich_stage_dir = outputs.generate_stage_dir(
+            output_dir,
+            site_dir.parent.name,
+            site_dir.name,
+            PipelineStage.ENRICH,
+        )
```

Comment:
the relevant root directory (e.g., `out/al/arcgis/enriched`) will not exist the first time this is run, but we assume it does exist within `caching` (we try to open write to `out/al/arcgis/enriched/.api_cache.tar.gz`).

Should we create the relevant directory first here?

Reply:
Good point, I'll fix this

Reply:
Fixed